src/db/migrations/20230703010356_add-onerep-profiles-table.js

```diff
@@ -2,16 +2,28 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
 export function up(knex) {
   return knex.schema.createTable("onerep_profiles", (table) => {
     table.increments("id").primary();
     table
       .integer("onerep_profile_id")
       .references("subscribers.onerep_profile_id")
       .nullable();
+    // @ts-ignore TODO: Determine if the following line should be changed
```

Comment:
Curious why this doesn't work, `varchar` should be valid no?

Reply:
It looks like `varchar` is implemented, but missing from the type [TableBuilder](https://github.com/knex/knex/blob/master/types/index.d.ts#L2420). I think it is recommended to use `.string()`.

Reply:
Ah OK, yeah "under the hood" in Postgres I believe TEXT and VARCHAR types are implemented the same way so `.text()` is probably what you want https://knexjs.org/guide/schema-builder.html#text